code/game/machinery/porta_turret/portable_turret.dm

```diff
@@ -41,7 +41,7 @@
 	var/locked = TRUE			//if the turret's behaviour control access is locked
 	var/controllock = FALSE		//if the turret responds to control panels
 
-	var/installation = /obj/item/gun/energy/e_gun/turret		//the type of weapon installed by default

```

Comment:
why not set installation to null on ai turrets instead of making it for all the turrets

Reply:
It's kinda a moot point since the only (pre-made) turret that uses it is the AI one but to avoid future errors I've changed it anyway.